src/renderer/helpers/api/local.js

```diff
@@ -609,8 +609,8 @@ export function parseLocalListVideo(video) {
     author: video.author.name,
     authorId: video.author.id,
```

Comment:
Movies don't have a channel ID on YouTube in search results, just a channel name, so this becomes `N/A` resulting in invalid channel links getting generated.

```suggestion
    video.author.id !== 'N/A' ? video.author.id : null,
```

You'll also have to update `ft-list-video` with a `v-else-if`, so that it shows the channel name without a link, if there is a channel name but no ID (both can be missing too, for example in shorts only playlists, which is why you can do just `v-else`).